packages/color-kit-block/src/Palette.tsx

```diff
@@ -1,17 +1,30 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { Text } from '@frontify/fondue';
+import { Text, merge } from '@frontify/fondue';
 
-import type { PaletteProps } from './types';
+import { EmptyView } from './EmptyView';
 import { Color } from './Color';
+import type { PaletteProps } from './types';
 
 export const Palette = ({ palette, isEditing }: PaletteProps) => {
     const { name, colors = [] } = palette;
 
+    if (colors.length === 0) {
+        return <EmptyView paletteName={`${name || 'Unitled Palette'} (no colors added)`} />;
+    }
+
     return (
-        <div data-test-id="palette" className="tw-flex tw-flex-col tw-space-y-2">
+        <div data-test-id="palette" className="tw-flex tw-flex-col tw-space-y-2 tw-mb-3 last:tw-mb-0">
             <Text color="x-weak">{name}</Text>
-            <div className="tw-flex tw-flex-wrap [&>div]:tw-h-6 [&>div:first-child>div]:tw-shadow-inner-line-first [&>div:last-child>div]:tw-shadow-inner-line-last">
+
+            <div
+                className={merge([
+                    'tw-flex tw-flex-wrap',
+                    isEditing
+                        ? '[&>div:first-child>div]:tw-shadow-inner-line-first [&>div:last-child>div]:tw-shadow-inner-line-last'
+                        : '[&>div:first-child>div>div>div]:tw-shadow-inner-line-first [&>div:last-child>div>div>div]:tw-shadow-inner-line-last',
```

Comment:
`[&>div:first-child>div>div>div]` :D

Reply:
I can explain in a call, this is the only way 😭

Reply:
I was just wondering if you could take it within the `<Color/>` component, such that it's clearer where you want to have your style :D. Probably you need to change style of the fondue component `Tooltip`.?

Reply:
Exactly, I cannot access the element I want, and we cannot add classes to `Tooltip` so this is what we get 😄

Reply:
fair enough 😄
maybe you could add a <div> around the <Tooltip/> component with this styling such that it's clearer where it belongs?

Reply:
I don't see the message, add a what? 😄

Reply:
wtf 😂 Now it's fixed and you should see it